pkg/cachemanager/cachemanager_test/cachemanager_integration_test.go

```diff
@@ -35,92 +35,137 @@ const (
 
 var cfg *rest.Config
 
+var (
+	configMapGVK = schema.GroupVersionKind{Group: "", Version: "v1", Kind: "ConfigMap"}
+	podGVK       = schema.GroupVersionKind{Group: "", Version: "v1", Kind: "Pod"}
+
+	configInstancOne      = "config-test-1"
+	configInstancTwo      = "config-test-2"
+	nsedConfigInstanceOne = "default/config-test-1"
```

Comment:
What is "nsed"?

Reply:
"namespaced"? I'd just write "namespaced"